UT4MasterServer/Controllers/ErrorsController.cs

```diff
@@ -20,10 +21,14 @@ public IActionResult Index()
 		{
```

Comment:
Can you also add logging exception here? `logger.LogError(ex)`

Reply:
Added, here is an example:
![image](https://user-images.githubusercontent.com/12861786/212544167-fbb682cd-e57f-4bcf-bd54-0a1db5c79ea9.png)